tests/test_cli.py

```diff
@@ -110,16 +105,12 @@ def test_cli_invalid_version(capsys, invalid_version):
     cli._setup_settings()
 
     matcher = re.compile(
-        r"^INFO    : aqtinstall\(aqt\) v.* on Python 3.*\n"
+        #  r"^INFO    : aqtinstall\(aqt\) v.* on Python 3.*\n"
```

Comment:
Why commented out not removed?

Reply:
Oh, I've forgotten to restore the version message.